epub33/overview/index.html

```diff
@@ -165,15 +169,15 @@ <h2>Package Document</h2>
 				<p>The Package Document also includes a <a data-cite="epub-33#sec-collection-elem"
 							><code>collection</code> element</a> [[EPUB-33]], which allows grouping of logically related
 						<a>Publication Resources</a>. This element exists to enable the development of specialized
-					content identification, processing and rendering features, such as the ability to define embedded
+					content identification, processing and rendering features such as the ability to define embedded
 					preview content, or assemble an index or dictionary from its constituent XHTML Content
```

Comment:
The comma after "content" seems superfluous.
```suggestion
					preview content or assemble an index or dictionary from its constituent XHTML Content
```